scripts/macro-support.js

```diff
@@ -99,67 +100,128 @@ export default class MacroSupport {
    * @param {number} chanceOfDamagedItems A number between 0 and 1 that corresponds to the percent chance an item will be damaged
    * @param {number} damagedItemsMultiplier A number between 0 and 1 that will lower a damaged items value
    * @param {boolean} removeDamagedItems If true, damaged items will be removed from the token rather than marked as damaged
+   * @param {string} mode e.g. "itempiles", "lootsheet"
    */
   convertSelectedTokensToLoot(
-    chanceOfDamagedItems,
-    damagedItemsMultiplier,
-    removeDamagedItems
+    {
+      token,
+      mode = "lootsheet",
```

Comment:
I don't love this `mode` concept. I think it would be better to have two separate methods that could utilize some shared behavior.

- `convertSelectedTokensToLootSheet`
- `convertSelectedTokensToItemPile`

Reply:
done